test/examples/config.js

```diff
@@ -6,12 +6,20 @@
  * Side Public License, v 1.
  */
 
-import path from 'path';
+import path, { resolve } from 'path';
 import { services } from '../plugin_functional/services';
+import fs from 'fs';
+import { KIBANA_ROOT } from '@kbn/test';
 
 export default async function ({ readConfigFile }) {
   const functionalConfig = await readConfigFile(require.resolve('../functional/config'));
 
+  // Find all folders in /examples and /x-pack/examples since we treat all them as plugin folder
```

Comment:
Can you expand on this? Why do we need to include example plugins from both when running the tests here?

Reply:
`x-pack/examples` basically expand `examples/`: the example app itself is in `examples/` (`examples/developer_examples`)

Also some plugins in `examples/`use basic x-pack features. They just don't have oss tests to call it out. Goot that it isn't really an issue anymore